### PR TITLE
Posting

### PR DESCRIPTION
**Pull request by LauraMakesArt**
This could get some additions to it as we go along, but to me this basic stuff would be better described in a video. If someone wants to build on this, go for it.

### CODE REVIEW

File: content/help/posting/index.md [hunk 1]
```diff
@@ -10,5 +10,58 @@ menu:
     identifier: posting
     weight: 18
 ---
+## Overview
 
-## Coming soon!
\ No newline at end of file
+"Posting" or commenting on a discussion is the primary way users will interact with your community. There is a range of discussion types, helping communities adjust the functionality to suit their needs.
+
+At the bottom of each thread, the user will encounter a text field:
```
Comment (linc) [inline content/help/posting/index.md]: thread => discussion

**Comment by linc** [comment]
Be careful about nomenclature to keep it consistent. It's always a "discussion" in Vanilla, never a "thread".

**Comment by LauraMakesArt** [comment]
@linc Should we perhaps make a small glossary somewhere on the back end? The writer in me wants varied language so that the sentences read more interestingly, but unified terms are for the best. Everything from "text field" to "cogwheel menu" are currently things I pull out of thin air, old docs, and blog posts, but I feel like you and todd should be setting the standards.

**Comment by linc** [comment]
I mean... yes... but maybe today we could just not say "thread" as that is verboten in our lingo.

**Comment by LauraMakesArt** [comment]
Mostly just throwing the idea out there for later. I've updated the doc.

**Pull request merged**